src/new.spec.js

```diff
@@ -111,5 +111,55 @@ describe("withStore", () => {
 
       expect(getState().baz).toBe("baz");
     });
+
+    it("are not called when its state/props dependencies do not change", async () => {
+      const baz = jest.fn(({ qux }, { bar }) => bar * qux);
```

Comment:
I think we should call it `sum` and it's dependencies `a` and `b` (or `i`/`j` or `n`/`m`), it will make the test either to read/understand.